base/musl/Dockerfile

```diff
@@ -73,3 +73,5 @@ COPY backport-ipmicomm.patch .
 COPY caputlog-waveform-fix.patch .
 COPY install_modules.sh .
 RUN NEEDS_TIRPC=YES ./install_modules.sh
+
+COPY build-static-ioc.sh /usr/local/bin
```

Comment:
Other similar scripts have been prefixed with `lnls-`, and had their `.sh` prefix dropped when installing. I'd rather keep this consistent.